python/mxnet/gluon/data/text/_constants.py

```diff
@@ -0,0 +1,26 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# coding: utf-8
+
+"""Constants relevant to text processing."""
+
+UNK_TOKEN = '<unk>'
+
+BOS_TOKEN = '<bos>'
```

Comment:
Isn't this usually <sos>?

Reply:
> Isn't this usually `<sos>`?
I see more use of `<bos>`, though honestly it would just be a preference, as long as we can agree on one.